i-Code-Doc/benchmarker/__init__.py

```diff
@@ -0,0 +1,41 @@
+from benchmarker.__version__ import __version__
+
+from benchmarker.config.benchmarker_config import (
+    T5BenchmarkerConfig,
+)
+from core.models import (UdopDualForConditionalGeneration,
+                         UdopUnimodelForConditionalGeneration,
+                         UdopConfig,
+                         UdopTokenizer,
+                        )
+
+__all__ = ['__version__']
+
+MODEL_CLASSES = {
+    "UdopUnimodel": {
+        "config": UdopConfig,
+        "config2d": UdopConfig,
+        "tokenizer": UdopTokenizer,
+        "model_attr_name": "UdopUnimodel",
+        "wordgap_data_converter": None,
+        "pretraining": UdopUnimodelForConditionalGeneration,
+        "token_classification": UdopUnimodelForConditionalGeneration,
+    },
+    "UdopDual": {
+        "config": UdopConfig,
```

Comment:
seems like UDOP and UDOP dual are using the same config class?

Reply:
This is how the config is initially design I remembered. Because the parameters are mostly shared.